### PR TITLE
Edit, delete post

### PR DESCRIPTION
**Pull request by BuiPhuongThaob**
## Related Tickets:
- [Task #27544](https://edu-redmine.sun-asterisk.vn/issues/27544)
- [Task #27546](https://edu-redmine.sun-asterisk.vn/issues/27546)

## Notes Impacted Areas:
- N/A.

### CODE REVIEW

**Comment by BuiPhuongThaob** [comment]
ready

File: resources/views/pages/profiles/mypost.blade.php [hunk 1]
```diff
@@ -4,29 +4,48 @@
...
+                                    @csrf
+                                    @method('DELETE')                                    
+                                    <button type="submit" class="btn" id="delete-btn"><i class="fas fa-trash-alt"></i>{{ trans('profile.delete') }}</button>
+                                </form>
+                            </div>                           
                         </div>
-                        <a href="#" class="col-lg-12">
-                            <p>{{ $post->intro }}</p>
+                        <a href="{{ route('posts.show', $post->id) }}" class="col-lg-12">
+                                <p>{{ $post->intro }}</p>
```
Comment (phamviethieu) [inline resources/views/pages/profiles/mypost.blade.php]: indent 4 spaces

File: app/Http/Controllers/PostController.php [hunk 2]
```diff
@@ -82,7 +82,10 @@ public function store(PostRequest $request)
      */
     public function show($id)
     {
-        //
+        $post = Post::find($id);
+        $provinces = Province::all();
+
```
Comment (huyenttm-0901) [inline app/Http/Controllers/PostController.php:87]: nếu ko tìm thấy post thì sao e?

File: resources/views/pages/edit_post.blade.php [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                <input 
+                    value="{{ $post->place->place_name }}"
+                    name="place"
+                    type="text" 
+                    class="form-control input_post" 
+                    placeholder="{{ trans('post.place_review') }}">
+                    <label for="prov_list">
+                        <select id="prov_list" name="prov_list" class="select_prov mdb-select md-form">
+                            <option value="{{ $post->place->province->id }}" selected>{{ $post->place->province->prov_name }}</option>
+                            @foreach($provinces as $prov)
```
Comment (huyenttm-0901) [inline resources/views/pages/edit_post.blade.php]: convention

File: resources/views/pages/profile.blade.php [hunk 4]
```diff
@@ -27,7 +27,7 @@ class="avt-image rounded-circle" alt="{{ trans('profile.user_avt') }}" id="avt-i
     <div class="">
         <ul class="nav nav-tabs tabs-alt justify-content-center">
             <li class="nav-item">
-                <a class="nav-link py-4 active" id="post" href="#">{{ trans('profile.post') }}</a>
+                <a class="nav-link py-4 active" id="post" href="{{route('posts.index') }}">{{ trans('profile.post') }}</a>
```
Comment (huyenttm-0901) [inline resources/views/pages/profile.blade.php]: convention

File: resources/views/pages/single_post.blade.php [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                <div class="comments-area">
+                    <h4>{{ trans('profile.comment') }}</h4>
+                    <div class="comment-list">
+                        
+                    </div>         
+                </div>               
+            </div>
+            <div class="col-lg-4">
+                <div class="blog_right_sidebar">
+                        <aside class="single_sidebar_widget post_category_widget">
```
Comment (huyenttm-0901) [inline resources/views/pages/single_post.blade.php]: indent 4 spaces

**Comment by BuiPhuongThaob** [comment]
ready

File: resources/views/pages/edit_post.blade.php [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+        </div>
+    </div>
+</form>
+@endsection
+
+@section('script')
+    <script src="{{ asset('bower_components/review_travel/js_travel/post.js') }}"></script>
+    <script src="{{ asset('bower_components/select2/dist/js/select2.min.js') }}"></script>
+    <script src="{{ asset('bower_components/review_travel/js_travel/select.js') }}"></script>
+    <script src="{{ asset('js/all.js') }}"></script>
```
Comment (hoangminhuet) [inline resources/views/pages/edit_post.blade.php]: nếu đã mix thì dùng như này nhé chị
```suggestion
    <script src="{{ mix('js/all.js') }}"></script>
```

File: resources/views/pages/single_post.blade.php [hunk 7]
```diff
@@ -0,0 +1,91 @@
+@extends('home')
+@section('css')
+    <link rel="stylesheet" href="{{ asset('css/all.css') }}">
```
Comment (hoangminhuet) [inline resources/views/pages/single_post.blade.php]: tương tự

File: resources/views/pages/single_post.blade.php [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+@yield('profile')
+@endsection
+@section('script')
+    <script src="{{ asset('js/all.js') }}"></script>
```
Comment (hoangminhuet) [inline resources/views/pages/single_post.blade.php]: tương tự

File: resources/views/pages/edit_post.blade.php (hunk 6 above)
Comment (BuiPhuongThaob) [reply resources/views/pages/edit_post.blade.php]: okie, cam on hoang nhe :3

**Comment by BuiPhuongThaob** [comment]
ready

**Pull request merged**